contrib/debug-tools/docs/remote-debug-pai-job.md

```diff
@@ -0,0 +1,85 @@
+# 1. Overview
+
+OpenPai remote debug tool only support **python** job. It contains two debug modes:
+
+1. Start remote debug when job task starts
+2. Inject breakpoint in source code and start debug when hit the breakpoint.
+
+Currently this feature is in experiment phrase, and we don't have a mechanism to notify user weather debug server is starts. Users need to check log to my theirself to get this notification.
+
+We recommend to use first approach to start debug. To use second approach, user need to manage the debug ports by themselves.
+
+***NOTICE: This is an experiment feature and may be changed in future release***
+
+# 2. Edit the job submission yaml to support debug
+To support the live debug, we need to modify the job submit yaml. Please request one debug port in `resourcePerInstance` field.
```

Comment:
Avoid to use "we". Use "users"

Reply:
`resourcePerInstance` -> taskRoles->[task role]->resourcePerInstance->ports

Reply:
fixed